src/components/shared-spatial-scatterplot/AbstractSpatialOrScatterplot.js

```diff
@@ -178,8 +178,9 @@ export default class AbstractSpatialOrScatterplot extends PureComponent {
     const { gl, tool } = this.state;
     const layers = this.getLayers();
 
-    const showCellSelectionTools = this.cellsLayer !== null || layerProps.findIndex(l => l.type === 'cells') > 0;
-    const showPanTool = this.cellsLayer !== null || layerProps.findIndex(l => l.type === 'bitmask') > 0;
+    const showCellSelectionTools = this.cellsLayer !== null
+      || (this.cellsEntries.length && this.cellsEntries[0][1].xy);
+    const showPanTool = this.cellsLayer !== null || layerProps.findIndex(l => l.type === 'bitmask' || l.type === 'raster') >= 0;
```

Comment:
How about if there is only a molecules or neighborhoods layer? Can we always show the pan tool? Is there a case when it would be unnecessary?

Reply:
Previously it was just `this.cellsLayer !== null` so I just built off that: https://github.com/vitessce/vitessce/blob/3393dba4730f9d379961ce1f56b09e40918d7a28/src/components/shared-spatial-scatterplot/AbstractSpatialOrScatterplot.js#L129-L130

Reply:
I think perhaps what I have is too aggressive.  I could probably leave the pan tool off like it used to be if we don't have a `cellsLayer` but I'm also open to other options.